### PR TITLE
lightning: add function `getChunkCompressedSizeForParquet` for solving parquet oom issue

### PR DESCRIPTION
**Pull request by lyzx2001**
### What problem does this PR solve?

Issue Number: ref #49234 

Problem Summary:

Lightning encounters OOM when importing Parquet files.

### What changed and how does it work?

Added the function `getChunkCompressedSizeForParquet`. Solution for the OOM issue will be pushed in later PRs.

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/lightning/importer/table_import.go [hunk 1]
```diff
@@ -1172,6 +1179,39 @@ func (tr *TableImporter) postProcess(
...
+	chunk *checkpoints.ChunkCheckpoint,
+	store storage.ExternalStorage,
+) (int64, error) {
+	reader, err := mydump.OpenReader(ctx, &chunk.FileMeta, store, storage.DecompressConfig{})
+	if err != nil {
+		return 0, errors.Trace(err)
+	}
+	parser, err := mydump.NewParquetParser(ctx, store, reader, chunk.FileMeta.Path)
+	if err != nil {
+		return 0, errors.Trace(err)
```
Comment (D3Hunter) [inline br/pkg/lightning/importer/table_import.go:1193]: close reader when failed to new parser?

**Review by D3Hunter — COMMENTED** [review]
rest lgtm

File: br/pkg/lightning/importer/table_import.go (hunk 1 above)
Comment (lyzx2001) [reply br/pkg/lightning/importer/table_import.go:1194]: modified

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**